### PR TITLE
feat(templates): Add iOS template

### PR DESCRIPTION
**Pull request by francoischalifour**
This PR adds the InstantSearch iOS template using CocoaPods.

I made `libraryName` in `.template.js` optional as well in this PR (until #35 is implemented as dependency strategy).

This way, the InstantSearch version is not asked in the CLI.

## Preview

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "start": "node src/cli/index.js",
     "test": "jest ./src",
+    "test:e2e:installs": "jest './scripts/e2e-installs.test.js'",
```
Comment (samouss) [inline package.json]: Is there a reason to use single quotes rather than nothing?

File: packages/tasks/ios/install.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const process = require('process');
```
Comment (samouss) [inline packages/tasks/ios/install.js]: Why this file is in `packages/...` rather than `src/tasks/...` ?

Comment (samouss) [inline packages/tasks/ios/install.js]: Does this file is really used? Seems that we do the same logic in the template's tasks?

Comment (francoischalifour) [reply packages/tasks/ios/install.js]: You're right, this folder shouldn't be there anymore (rebased from the old architecture).

Comment (francoischalifour) [reply packages/tasks/ios/install.js]: Indeed, I'll reuse the tasks in `.template.js`.

File: package.json (hunk 1 above)
Comment (francoischalifour) [reply package.json]: I can remove them, it works without quotes now. I think it used to get processed as a regex before but not anymore.

**Pull request merged**